demos/JWST/S5_fit_par_template.epf

```diff
@@ -7,37 +7,65 @@
 # If U/LU, PriorPar1 and PriorPar2 represent upper and lower limits of the parameter/log(the parameter).
 # If N, PriorPar1 is the mean and PriorPar2 is the standard deviation of a Gaussian prior.
 #-------------------------------------------------------------------------------------------------------
-#
+
 # ------------------
-# ** Transit/eclipse parameters **
+# ** BATMAN/POET/Starry Planet Parameters **
 # ------------------
-rp           0.148          'free'         0.15         0.1          N
-#fp           0.008          'free'         0           0.5          U
+rp          0.148           'free'          0.15        0.1         N
+per         4.055259        'free'          4.055259     1e-5       N
+t0          57394.524       'free'          57394.524    0.05       N
+time_offset 0               'independent'
+inc         87.83           'free'          87.83        0.25       N
+a           10.9            'free'          10.9         1          N
+ecc         0.0             'fixed'         0            1          U
+w           90.             'fixed'         0            180        U
+# fp          0.008           'free'          0           0.5         U
+# t_secondary 0.25            'free'          0.25         0.05       N  
+# Rs          0.12            'independent'
+
 # ----------------------
-# ** Phase curve parameters **
+# ** Phase Curve Parameters **
 # ----------------------
-#AmpCos1      0.4           'free'         0            1            U
-#AmpSin1      0.01          'free'         -1           1            U
-#AmpCos2      0.01          'free'         -1           1            U
-#AmpSin2      0.01          'free'         -1           1            U
-# ------------------
-# ** Orbital parameters **
-# ------------------
-per          4.055259       'free'         4.055259     1e-5         N
-t0           57394.524      'free'         57394.524    0.05         N
-time_offset  0              'independent'
-inc          87.83          'free'         87.83        0.25         N
-a            10.9           'free'         10.9         1            N  # An incorrect a/R* seems to have been used in the simulations
-ecc          0.0            'fixed'        0            1            U
-w            90.            'fixed'        0            180          U
+# sinusoid_pc
+# AmpCos1      0.4           'free'         0            1            U
+# AmpSin1      0.01          'free'         -1           1            U
+# AmpCos2      0.01          'free'         -1           1            U
+# AmpSin2      0.01          'free'         -1           1            U
+# poet_pc
+# cos1_amp     0.4           'free'         0            1            U
+# cos1_off     0.00          'free'         0.4          0.6          U
+# cos2_amp     0.1           'free'         0            1            U
+# cos2_off     0.00          'free'         0.2          0.3          U
```

Comment:
It the units of these offset terms are days, aren't these quite restrictive and too "informative", especially for a demo/template? With the sinusoid terms, I've tried to make the priors minimally informative. And wouldn't units of degrees be more convenient so that you don't need to deal with different minimally informative priors for different orbital periods? With this as it is, I can imagine someone accidentally publishing prior-dominated results. Meanwhile, if the units are degrees then a prior of -180 to 180 for cos1_off and -90 to 90 (or 0 to 180 if the fits prefer an offset near 90) for cos2_off would be quick and easy priors for all planets

Reply:
The offsets used to have a prior of [-1, 1], but you said that was too large, so I shrunk them.  Now they're too small. What to do?!?!?  lol

An offset in degrees does make a lot of sense.

Reply:
Done!